libraries/testkit/syncgateway.py

```diff
@@ -13,7 +20,9 @@ def __init__(self, target):
         self.ansible_runner = AnsibleRunner()
         self.ip = target["ip"]
         self.url = "http://{}:4984".format(target["ip"])
+        self.admin_url = "http://{}:4985".format(target["ip"])
```

Comment:
perhaps we could have a self.admin = Admin(). Then everything routed to the admin url would be explicit in the test. 

for example
```
sg = SyncGateway()
sg.verify_launched()
sg.admin.create_db()
```